support-frontend/assets/components/subscriptionCheckouts/summary.jsx

```diff
@@ -21,9 +22,11 @@ export type DataListItem = {
 
 type PropTypes = {
   billingPeriod: BillingPeriod,
-  changeSubscription?: string | null,
+  changeSubscription?: Option<string>,
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Are these props actually used or not?

Reply:
They are used, I'm sure why flow has a problem with them

Reply:
It's lint that has the problem rather than Flow, I've just tried this and it works fine without disabling those validations for me